cli/keys_test.go

```diff
@@ -128,12 +127,12 @@ func TestKeysCmd_RandomPassphrase(t *testing.T) {
 
 	t.Run("no passphrase prompt when random-passphrase flag is used", func(t *testing.T) {
 		ctrl := gomock.NewController(t)
-		prompt := mock_prompts.NewMockPrompt(ctrl)
+		prompt := prompter.NewMockPrompter(ctrl)
 		defer ctrl.Finish()
 
 		prompt.
 			EXPECT().
-			Passphrase().
+			InputSecret(gomock.Any()).
```

Comment:
Isn't it `prompt.InputSecret()` ?

Reply:
`InputSecret` is an input with hidden content like `Passphrase`. But with this new approach `InputSecret` receives the message as an argument like [here](https://github.com/NethermindEth/sedge/blob/refactor/cli/cli/keys.go#L83).

Reply:
I'm talking about the pkg of the function. That may work because the mock is in the same package as the test, but remember we moved all the mocks to an independent package; therefore, you will have to update this, and move the mock to that package.

Reply:
Use this pattern for the mock:
https://github.com/NethermindEth/sedge/blob/develop/cli/actions/gen.go

Reply:
Done here https://github.com/NethermindEth/sedge/pull/228/commits/284d9ad6e7276e2912568fb5bf8d229214144f7c